msteams-platform/m365-apps/extend-m365-meeting-app.md

```diff
@@ -0,0 +1,191 @@
+
+1. Select the sample tab configuration and select **Save**. The app is available in the meeting chat view.
+
+    :::image type="content" source="images/teams-meeting-chat-view.png" alt-text="{alt-text}":::
+
+# [Extend an existing Teams meeting app to Outlook](#tab/existing-app)
+
+To extend your existing Teams meeting app in Outlook, ensure the following:
+
+> [!div class="checklist"]
+>
+> * [Update your app manifest](#update-your-app-manifest).
+> * [Use Microsoft Teams JavaScript client library (TeamsJS) v2.5.0 or later](#use-teamsjs-version-250-or-later).
+> * [Configure Content Security Policy headers](#configure-csp-headers-for-outlook).
+> * [Update Microsoft Entra app registration for single sign-on (SSO)](#update-microsoft-entra-app-registration-for-sso).
+> * [Upload your updated app in Teams Admin Center](#upload-your-custom-app-in-teams-admin-center).
+
+If you have an existing meeting app, make a copy or a branch of your production project and update your app `id` in the app manifest to use a new identifier that is different from the production app ID for testing.
+
+## Update your app manifest
+
+Update the [app manifest](~/resources/schema/manifest-schema.md) (previously called Teams app manifest) schema version to 1.13 or later.
+
+Open your Teams app manifest and update the `$schema` and `manifestVersion` properties as follows:
+
+```json
+{
+    "$schema" : "https://developer.microsoft.com/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+    "manifestVersion" : "1.16"
+}
+```
+
+### Use TeamsJS version 2.5.0 or later
+
+Use TeamsJS client library version 2.5.0 or later to extend your Teams meeting app to run in Outlook. If your app uses TeamsJS v2.5.0 or later, application code changes aren't required to run in Outlook.
+
+If you need to upgrade your TeamsJS npm package from v1.x.x to the latest v2.x.x TeamsJS, you can use Teams Toolkit to help identify and automate the required code changes. For more information, see [TeamsJS library](../tabs/how-to/using-teams-client-library.md#whats-new-in-teamsjs-version-2xx).
+
+Following are the steps to update your TeamsJS to the latest version using Teams toolkit:
+
+1. Open your app in Visual Studio Code and select **Teams Toolkit** extension.
+1. Under **View**, select **Command Palette...** or **Ctrl+Shift+P**.
+1. Run the command `Teams: Upgrade Teams JS SDK and code references`.
+
+Your package.json file references `@microsoft/teams-js@2.x.x` and your `*.js/.ts` and `*.jsx/.tsx` files are updated with:
+
+* Import statements for `teams-js@2.x.x`.
+* [Function, Enum, and Interface calls](../tabs/how-to/using-teams-client-library.md#whats-new-in-teamsjs-version-2xx) for `teams-js@2.x.x`.
+* `TODO` comment reminders flagging areas that might be impacted by [Context](../tabs/how-to/using-teams-client-library.md#updates-to-the-context-interface) interface changes.
```

Comment:
Update: apologies, I saw the context in the Teams channel post thread

I think this may lead to some confusion though... perhaps quotations can help communicate, please let me know what you think:
```suggestion
* New "`TODO`" comment reminders flagging areas that might be impacted by [Context](../tabs/how-to/using-teams-client-library.md#updates-to-the-context-interface) interface changes.
* New "`TODO`" comment reminders to [convert callback functions to promises](../tabs/how-to/using-teams-client-library.md#callbacks-converted-to-promises).
```